client/tsconfig.json

```diff
@@ -17,6 +17,7 @@
     ],
     "useDefineForClassFields": false
   },
+  "exclude": ["./cypress.config.ts"],
```

Comment:
By the way - this was important so that the view in VS Code doesn't claim we have a bunch of errors even when the tests are working